coderd/coderd.go

```diff
@@ -1057,6 +1057,9 @@ func New(options *Options) *API {
 			r.Get("/rich-parameters", api.templateVersionRichParameters)
 			r.Get("/external-auth", api.templateVersionExternalAuth)
 			r.Get("/variables", api.templateVersionVariables)
+			r.Route("/presets", func(r chi.Router) {
```

Comment:
Nit: do we need a `Route` now since it's just a single endpoint?

Reply:
good catch. removing.